nodewatcher/settings.py

```diff
@@ -354,6 +357,7 @@ def user_url(user):
 
     # Accounts support.
     'nodewatcher.extra.accounts',
+
```

Comment:
Not for the blank line, I just asked you to move what was previously there above? Please remove the blank line.